biomage/rds/migrate.py

```diff
@@ -0,0 +1,83 @@
+import os
+import subprocess
+from pathlib import Path
+
+import click
+
+from ..utils.AuroraClient import AuroraClient
+from ..utils.constants import DEFAULT_AWS_ACCOUNT_ID, DEVELOPMENT, STAGING
+
+# Assuming that biomage-utils and iac root folders are located in the same folder
+MODULE_PATH = os.path.dirname(os.path.abspath(__file__))
+DEFAULT_IAC_PATH = os.path.join(MODULE_PATH, "../../../iac")
+IAC_PATH = os.getenv("BIOMAGE_IAC_PATH", DEFAULT_IAC_PATH)
+
+
+def _migrate(iac_path, migration_env):
+    proc = subprocess.Popen(
+        ["node_modules/.bin/knex", "migrate:latest", "--cwd", iac_path],
+        cwd=iac_path,
+        env=migration_env,
+    )
+    proc.wait()
+
+
+@click.command()
+@click.option(
+    "-i",
+    "--input_env",
+    required=True,
+    default=DEVELOPMENT,
+    show_default=True,
+    help="Path to the IAC folder",
+)
+@click.option(
+    "-s",
+    "--sandbox_id",
+    required=False,
+    default=None,
+    show_default=True,
+    help="Sandbox id to migrate to. Required if migrating to staging",
+)
+@click.option(
+    "--iac_path",
+    required=False,
+    default=IAC_PATH,
+    show_default=True,
+    help="Path to the IAC folder",
+)
+def migrate(iac_path, sandbox_id, input_env):
+    """
+    Migrate database in environment
```

Comment:
change also the comment to reflect the command structure maybe? like in biomage rds run (or maybe there's another way to document it

Reply:
Comment updated